### ISSUE TITLE
User cannot use keyboard to navigate through user settings/profile button (Firefox)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9184]
## Observed behavior
Using a keyboard, the focus ring gets stuck on "Profile" in dropdown.

![2022-03-14 12 02 56](https://user-images.githubusercontent.com/13563002/158243024-31e6b4f9-208b-4331-a359-a194690409eb.gif)

## Errors and logs
None

## Expected behavior
User should be able to navigate through all options (unless we don't want keyboard users to ever sign out).

## User-facing consequences
Users are unable to navigate to other profile settings.

## Steps to reproduce
1. Sign in
2. Use the keyboard to navigate to the user settings button/dropdown
3. Attempt to use the keyboard to navigate through that dropdown

## Context
Kolibri 0.15
MacOS
Firefox

**Comment by neoshri** [comment]
@sairina hey I want to contribute to the issue how can get started

**Comment by sairina** [comment]
Hi there @neoshri - to get started, after forking the repo follow the directions in our [dev documentation](https://kolibri-dev.readthedocs.io/en/develop/) to get Kolibri set up locally and the development server working.
- You'll want to work off our `release-v0.15.x` branch
- Following the "Steps to reproduce" should allow you to see the issue here

Let me know if you have any more specific questions.

**Comment by marcellamaki** [comment]
Possible mac settings issue 

![profile-dropdown.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/d4851125-c899-4158-8467-633c21e069f9)

**Comment by marcellamaki** [comment]
cc @radinamatic -- to double check before closing

**Comment by pcenov** [comment]
Confirmed - the navigation works correctly on Safari, Firefox and Chrome on Mac after enabling the correct settings.
